style/index.js

```diff
@@ -132,6 +132,7 @@ const infoPois = [
 ];
 
 const nameWithEle = "[name] + '\n' + [ele]";
+const nameWithSmallerEle = (eleSize) => '[name] + "\n"<<Format size="'+(eleSize)+'">>[ele]<</Format>>'
```

Comment:
Paramtrizoval by som az ked ten parameter budeme realne potrebovat. Inac len zbytocne (mierne) zvysuje narocnost pochopenia stylu.

Reply:
Alebo mozno aj nie, ale ked to dam do extension tak to mozno zjednodusi.